zebra-network/src/peer_set/candidate_set.rs

```diff
@@ -133,12 +126,20 @@ where
             address_book,
             peer_service,
             next_peer_min_wait: sleep(Duration::from_secs(0)),
+            next_update_time: Instant::now(),
         }
     }
 
     /// Update the peer set from the network, using the default fanout limit.
     ///
     /// See [`update_initial`][Self::update_initial] for details.
+    ///
+    /// ## Security
```

Comment:
Let's delete this repeated comment - otherwise they'll get out of sync, and that could be confusing or lead to security bugs.